### ISSUE TITLE
t.Numeric causing UNKNOWN errors when validating with options. undefined is not an object (evaluating 'schema.$id')

### ISSUE THREAD

**Issue by Barbiero** [elysiajs/elysia#487]
### What version of Elysia.JS is running?

0.8.17

### What platform is your computer?

Linux 5.15.146.1-microsoft-standard-WSL2 x86_64 x86_64

### What steps can reproduce the bug?

minimum code for reproduction:

```ts
import { Elysia, t } from "elysia";

const app = new Elysia()
  .get("/:id/test", ({ params: { id } }) => id, {
    params: t.Object({
      id: t.Numeric({ minimum: 0 }),
    }),
    error(err) {
      if (err.code === "VALIDATION") {
        const idErr = err.error.all.find((err) => err.path === "/id")!;
        console.error(idErr);
        return "id must be a non-negative number";
      }
      console.error(err.code);
    },
  })
  .listen(3000);

console.log(`🦊 Elysia is running at ${app.server?.hostname}:${app.server?.port}`);
```

run this server, and then try the following requests:

|Request | Expected result | Actual Result |
|---|---|---|
|http://localhost:3000/0/test | 0 | 0 |
| http://localhost:3000/SomeString/test | id must be a non-negative number | id must be a non-negative number |
| http://localhost:3000/-1/test | id must be a non-negative number | **undefined is not an object (evaluating 'schema.$id')** |

### What is the expected behavior?

I expected that `t.Numeric` would fail with a `VALIDATION` code when the `id` param did not pass the `minimum` option part of the schema validation.

### What do you see instead?

We get an `UNKNOWN` error, and a weird error message that I see no helpful documentation around.

### Additional information

This only happens for inner properties that fail validation for the number value; for example, `{ multipleOf: 1 }` will fail with the same problem if we pass an id of `0.5`.

**Comment by SaltyAom** [comment]
Fixed in [711925a](https://github.com/elysiajs/elysia/pull/465/commits/711925a5b5561452b58979b26917cb13a6cf8d9c) under `1.0.0-beta.14`.

If resolved, please leave the issue opened until release on next stable release.

**Comment by SaltyAom** [comment]
Should be fixed in 1.0.0

Feels free to reopen if issue still persists.
